admin/class-convertkit-mm-admin.php

```diff
@@ -142,7 +143,33 @@ public function register_settings() {
 						'tags' => $tags,
 					)
 				);
+			}
+		}
 
+		// Output product to tag mappings.
+		if ( empty( $products ) ) {
+			add_settings_field(
+				'convertkit-empty-mapping',
+				apply_filters( $this->plugin_name . '_display_convertkit_mapping_product', esc_html__( 'Mapping', 'convertkit-mm' ) ),
+				array( $this, 'display_options_empty_mapping_products' ),
+				$this->plugin_name,
+				$this->plugin_name . '-ck-mapping'
+			);
+
+		} else {
+			foreach ( $products as $key => $name ) {
+				add_settings_field(
+					'convertkit-mapping-product-' . $key,
```

Comment:
This is another one that's probably a good candidate for a constant (and not an env var)

Reply:
Agreed - I'd like to move settings to their own class with a constant as we do in the main ConvertKit plugin, and will do this in another PR.